DESCRIPTION

```diff
@@ -0,0 +1,18 @@
+Package: vitalsigns
+Title: A Utility Package for Conservation International's Vital Signs Project 
+Version: 0.1
+Authors@R: person("Thomas", "Johnson", email = "thomascjohnson@gmail.com",
```

Comment:
I would add "SystemRequirements" field though that mentions the AWS CLI dependency. See https://github.com/azvoleff/teamlucc/blob/master/DESCRIPTION#L45-L48 for one example.